scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -237,6 +237,48 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
         getBlockPosition: function(block) {
             return _.indexOf(vm.blocks, block);
         },
+        splitCurrentTextBlockAndInsertBetween: function(textBlockCtrl, blockToInserts) {
+            var indexWhereToAddNewBlock = vm.getBlockPosition(textBlockCtrl.block) + 1;
```

Comment:
In my experience it is a better practice to use variable names as short and succinct as possible, without becoming to ambiguous. It makes it easier for programmers that are unfamiliar with your code to understand it. The reason for this being that long identifier names take a long time to read and a lot of 'brain space' to keep in mind. Whereas in something like literature it is nice for the words to be as expressive as possible, in programming I find it is better to make it easy for others to read the logic and flow of execution, as opposed to the clarity of identifier meanings.

For example, this whole component is mostly about 'block text', so it is a bit redundant to add 'Block' into every word.

In my opinion, it would look nicer as:

* `splitAndInsert` as opposed to `splitCurrentTextBlockAndInsertBetween`
* `idx` or `index` instead of `indexWhereToAddNewBlock`
* `after` instead of `textThatWasAfterCaret`
* `end()` instead of `extractEndOfBlock()`
* etc....

If any of the above prove to be slightly hard to follow, then an inline comment above the declaration can clarify their meaning.

This is just my personal view on declaring identifiers and I am happy for you to leave it as is if you disagree, but I encourage you to consider it. Readability is much more important than [brevity](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=define%20brevity) when it comes to programming.